README.md

```diff
@@ -136,61 +132,66 @@ There are four configs within the **Node Properties** group.
   * If TRUE the node will be deployed / redeployed when changes are detected.
   * If FALSE the node will not be deployed or will be dropped during redeployment.
 
-### Options
+<h3 id="looped-load-options">Looped Load Options</h3>
 
 ![Looped Load-options](https://github.com/coalesceio/Incremental-Nodes/assets/7216836/7df81cee-6ab8-478b-a60c-3c625cd2c927)
 
-* **Enable tests**:This toggle can be enabled to perform some data quality tests.
+* **Enable tests**: This toggle can be enabled to perform some data quality tests.
 * **Pre-SQL**: Any SQL to be executed as a predecessor to data insert operation can be mentioned here.
 * **Post-SQL**: Any SQL to be executed post the data insert operation can be specified here.
 
-### Load Options
+### Looped Load Load Options
 
 ![Incremental_load_options](https://github.com/coalesceio/Incremental-Nodes/assets/7216836/59f316de-c1a8-4e08-a028-dd31bfa4e092)
 
 * **Load Type**:This toggle helps you to choose different accepted Load Types from Config.If no load type pararmeter is specified in the workspace,then it falls back to the config level load type.
 
-### Record Selection Options
+### Looped Load Record Selection Options
 
 ![LL-RecordSelectionOptions](https://github.com/coalesceio/Incremental-Nodes/assets/7216836/7532bb77-b9da-4cf1-9d46-6334c020da44)
 
 * **Grouping Type**:
-  * Key Based:The incoming source data is grouped based on the bucket column as the key.
-  * Range Buckets:The incoming source data is grouped based on the bucket column into different  groups mentioned in 'Number of Buckets' textbox.
-    * Number of Buckets :The number of groups based on which data will be divided.
-* **Bucket Columns**:
+  * Key Based: The incoming source data is grouped based on the bucket column as the key.
+  * Range Buckets: The incoming source data is grouped based on the bucket column into different  groups mentioned in 'Number of Buckets' textbox.
+  * Number of Buckets: The number of groups based on which data will be divided.
+* **Number of Groups**: [Missing]
```

Comment:
@jgontarz @prj2001-udn There seems to be a definition missing here

Reply:
The missing information has been added